src/super_gradients/training/datasets/samplers/infinite_sampler.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
```

Comment:
please remove

Reply:
removed and added license in https://github.com/Deci-AI/super-gradients/pull/223/commits/207848406e975b66dfeb5eaa801a0047ebefb670